x-pack/plugins/observability_solution/logs_explorer/public/components/data_source_selector/sub_components/selector_footer.tsx

```diff
@@ -57,7 +57,7 @@ export const ESQLButton = (props: DiscoverEsqlUrlProps) => {
 
   return (
     <EuiFlexItem grow={false}>
-      <EuiButton {...linkProps} color="success" size="s" data-test-subj="esqlLink">
+      <EuiButton {...linkProps} color="accentSecondary" size="s" data-test-subj="esqlLink">
```

Comment:
@patpscal The latest guideline on buttons is to not use `accentSecondary` for buttons, afaik ([slack conversation](https://elastic.slack.com/archives/C073NM7NGLD/p1733939342192109))
Should this stay `success` (green)?

Reply:
This one is actually not a confirm button or anything similar, but a navigation link as a call to action? Should in that case be success anyway, and not a teal color as we had in the beginning?

Reply:
IMO, `primary` here represents something that would trigger an action related to the current selector (which is not the case):
<img width="535" alt="Screenshot 2024-12-12 at 12 22 24" src="https://github.com/user-attachments/assets/7c2a23aa-bf8a-42d4-9572-9c3cf96c588f" />

while using `accent` seems a bit off compared to the whole color set we used on the page and the selector:

<img width="531" alt="Screenshot 2024-12-12 at 12 21 29" src="https://github.com/user-attachments/assets/d25ab00c-4b90-410a-a319-6b987ce9fa0f" />

@patpscal any thoughts on this?

Reply:
I'm with @tonyghiani here about the `primary` usage as this is merely an option, not something we should encourage the user to enable in order to continue with the process (we guide users toward the important actions of the flow with our primary buttons) If anything, it feels more appropriate to use `text` (the gray one) or an `accent` if we want to highlight it without insinuating it's mandatory. Since this is Discover, I'll also wait for confirmation from Platform designers @ek-so @MichaelMarcialis

Reply:
Just a nit @patpscal this is not Discover but the Logs Explorer app, which is set to be removed in v9.

Reply:
Sounds good to me, it would also align with the button design used in discover for this CTA:
<img width="599" alt="Screenshot 2024-12-12 at 13 24 55" src="https://github.com/user-attachments/assets/3ebbc2f5-ced0-49a7-80e5-7a113ad3f930" />

Reply:
Addressed with [style(logs): update button color](https://github.com/elastic/kibana/pull/202746/commits/6d70a3ceca9171571653fe36196ca569cba45c86)

Reply:
@mgadewoll how does that sound?

Reply:
That works 🙂 Thanks for the responsiveness on this!

Reply:
@tonyghiani @patpscal There are 2 more usages of `accentSecondary` on buttons [here](https://github.com/elastic/kibana/pull/202746/files#diff-4514f99b9f1c0562ac8c404dd9cc0a596a9f1c0d1b6113cef5a4ef5852103019R86) and [here](https://github.com/elastic/kibana/pull/202746/files#diff-d1421f4e40d64f102d357c5ca1f7ef7d777b1b9906024e5fbb70bfa50067e02fR223).

Reply:
As the first one is a copy button and the second a link to explore the logs, I would pass both of them to `primary`, wdyt @patpscal ?

Reply:
Agreed @tonyghiani , let's turn them `primary`. Thank you for catching that @mgadewoll 🙏